### PR TITLE
Creating release task

### PR DESCRIPTION
**Pull request by lucis**
image

### CODE REVIEW

**Review by tlgimenes — APPROVED** [review]
AHTM. I'd just add this version to an exported variable so we can know which version of live is running. With this, we could do some "feature flags" depending on the live version

File: scripts/release.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { exec, OutputMode } from "https://deno.land/x/exec@0.0.5/mod.ts";
+import { increment } from "https://deno.land/std@0.167.0/semver/mod.ts";
+import { Select } from "https://deno.land/x/cliffy@v0.25.5/prompt/mod.ts";
+
+await exec("git fetch --tags");
+
+const response = await exec(`bash -c "git tag | sort -V | tail -1"`, {
+  output: OutputMode.Capture,
+});
```
Comment (igorbrasileiro) [inline scripts/release.ts:9]: This would break on windows, wouldn't it?

**Review by igorbrasileiro — APPROVED** [review]

File: scripts/release.ts (hunk 1 above)
Comment (lucis) [reply scripts/release.ts:9]: Can you try?

Comment (igorbrasileiro) [reply scripts/release.ts:9]: Sure!

**Pull request merged**